org.eclipse.lsp4e/plugin.xml

```diff
@@ -480,67 +480,67 @@
       <type name="org.eclipse.lsp4e.text"></type>
    </extension>
    <extension point="org.eclipse.ui.editors.markerAnnotationSpecification">
-		<specification annotationType="org.eclipse.lsp4e.read"
-			label="LSP Read Occurrence"
-			textPreferenceKey="LSP4EReadOccurrenceIndication" textPreferenceValue="false"
-			highlightPreferenceKey="LSP4EReadOccurrenceHighlighting"
-			highlightPreferenceValue="true" contributesToHeader="false"
-			overviewRulerPreferenceKey="LSP4EReadOccurrenceIndicationInOverviewRuler"
-			overviewRulerPreferenceValue="true"
-			verticalRulerPreferenceKey="LSP4EReadOccurrenceIndicationInVerticalRuler"
-			verticalRulerPreferenceValue="false" colorPreferenceKey="LSP4EReadOccurrenceIndicationColor"
-			colorPreferenceValue="212,212,212" presentationLayer="4"
-			showInNextPrevDropdownToolbarAction="true"
-			showInNextPrevDropdownToolbarActionKey="org.eclipse.lsp4e.read.showInNextPrevDropdownToolbarAction"
-            isGoToNextNavigationTarget="true"
```

Comment:
Could you please keep the line indention of the document. That makes it easier to track the changes and keeps the original formatting.

Reply:
Sorry, I think we have a misunderstanding. I mean the newly added lines should use the same format as the lines from the existing docuement (tab instead of spaces):
```java
   <extension point="org.eclipse.ui.editors.markerAnnotationSpecification">
		<specification annotationType="org.eclipse.lsp4e.read"
			label="LSP Read Occurrence"
			textPreferenceKey="LSP4EReadOccurrenceIndication" textPreferenceValue="false"
			highlightPreferenceKey="LSP4EReadOccurrenceHighlighting"
			highlightPreferenceValue="true" contributesToHeader="false"
			overviewRulerPreferenceKey="LSP4EReadOccurrenceIndicationInOverviewRuler"
			overviewRulerPreferenceValue="true"
			verticalRulerPreferenceKey="LSP4EReadOccurrenceIndicationInVerticalRuler"
			verticalRulerPreferenceValue="false" colorPreferenceKey="LSP4EReadOccurrenceIndicationColor"
			colorPreferenceValue="212,212,212" presentationLayer="4"
			showInNextPrevDropdownToolbarAction="true"
			showInNextPrevDropdownToolbarActionKey="org.eclipse.lsp4e.read.showInNextPrevDropdownToolbarAction"
			isGoToNextNavigationTarget="true"
			isGoToNextNavigationTargetKey="org.eclipse.lsp4e.read.isGoToNextNavigationTarget"
			isGoToPreviousNavigationTarget="true"
			isGoToPreviousNavigationTargetKey="org.eclipse.lsp4e.read.isGoToPreviousNavigationTarget"
			textStylePreferenceKey="LSP4EReadOccurrenceTextStyle"
			textStylePreferenceValue="NONE">
		</specification>
   </extension>
```

> Otherwise, mixing tabs and spaces leads to different indentation results depending on the IDE settings.

Yes, this should be done in a separate issue, I guess.

Reply:
I adapted the white space indentation characters to those previously used.